src/components/Game.jsx

```diff
@@ -1,37 +1,32 @@
-import React from "react"
+import React, { useState, useEffect } from "react"
 import { connect } from "react-redux"
 import { updateScore, updateRounds } from "../store/game"
 import { fetchPeople } from "../store/people"
 import Picture from "./Picture"
 
-export class Game extends React.Component {
-  constructor(props){
-    super(props)
-    this.state = {
-      selectedPerson: {},
-      renderOverlay: false,
-    }
-  }
+const Game = (props) => {
+
+  const { roundPeople, realPerson } = props;
+  const [ selectedPerson, setSelectedPerson ] = useState({})
+  const [ renderOverlay, setRenderOverlay ] = useState(false)
 
-  componentDidMount(){
-    const { fetchPeople } = this.props;
+  useEffect(() => {
+    const { fetchPeople } = props;
     fetchPeople();
-}
+  }, []);
 
-  pictureClickHandler = (id) => {
-    const { selectedPerson } = this.state;
-    const { roundPeople, updateScore, score, realPerson} = this.props;
+  let pictureClickHandler = (id) => {
+    // const { selectedPerson } = this.state;
+    const { roundPeople, updateScore, score, realPerson} = props;
 
     // When the user clicks on an image
```

Comment:
great job with the inline comments, they really help to understand what you're trying do